src/collector/Backend.cpp

```diff
@@ -54,8 +55,7 @@ void Backend::clone_from(const Backend & other)
 
     std::memcpy(&m_stat, &other.m_stat, sizeof(m_stat));
     std::memcpy(&m_calculated, &other.m_calculated, sizeof(m_calculated));
-
-    m_read_only = other.m_read_only;
+    m_status_text = other.m_status_text;
```

Comment:
I think it's easier and better to keep status text though.
1) `m_status_text` is maintained in the same way as for other objects (`Group`, `Couple`, ...). It's good to use common pattern;
2) Python mastermind shouldn't do any calculations in this place. @nobodyisme, what do you think?